state/block_test.go

```diff
@@ -0,0 +1,75 @@
+package state
+
+import (
+	"fmt"
+	"math/big"
+	"os"
+	"reflect"
+	"testing"
+
+	"github.com/ChainSafe/gossamer/common"
+	"github.com/ChainSafe/gossamer/core/types"
+)
+
+func TestGetBlockByNumber(t *testing.T) {
+	dataDir := "../test_data/block"
```

Comment:
Please clean up any test artifacts. Check out the top `cmd/gossamer/configcmd_test.go` for some copy pasta

Reply:
this same test is in the #503 , I've added comments there with snippets for the fix, could this be applied there instead ?

Reply:
It would be nice to apply here but I'm ok with it being addressed in #503.

Reply:
done